frontend/src/utils/focus-management.ts

```diff
@@ -36,7 +36,7 @@ const focusableSelector = [
   "textarea:not([disabled])",
 ]
   .map(
-    import.meta.env.NODE_ENV === "test"
+    import.meta.env?.NODE_ENV === "test"
```

Comment:
I wonder if this should use `import.meta.test` instead? Documented here: https://nuxt.com/docs/api/advanced/import-meta#runtime-app-properties

Not necessary for this PR, but if you think so, maybe an easy follow up issue for later on.

Reply:
Replaced 👍